kernel/src/analysis/tree_sitter.rs

```diff
@@ -84,7 +81,7 @@ pub fn get_query_nodes(
                 filename: filename.to_string(),
                 variables: variables.clone(),
             },
-        })
+        });
```

Comment:
Added the semicolon given that we're not returning. See [this](https://rust-lang.github.io/rust-clippy/master/index.html#/semicolon_if_nothing_returned).